internal/v1/handler_blueprints.go

```diff
@@ -69,6 +69,11 @@ func (h *Handlers) CreateBlueprint(ctx echo.Context) error {
 		return err
 	}
 
+	metadata, err := json.Marshal(blueprintRequest.Metadata)
```

Comment:
If there are no metadata, this results in `null` string (or rather byte array) https://go.dev/play/p/9o0KYX8FiNy
Are we ok with that, or would we rather have actual nil and NULL in the DB?

It's not a big deal really and can be handled in a follow-up.

Reply:
I slightly lean towards having `NULL` in the DB which is more efficient on storage.